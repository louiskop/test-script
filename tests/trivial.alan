source trivial
begin
	relax
end
