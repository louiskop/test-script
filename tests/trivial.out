'source'
Identifier: 'trivial'
'begin'
'relax'
'end'
